x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -741,6 +746,7 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = memo((props: Aler
             ref={dataGridRef}
             renderCustomGridBody={dynamicRowHeight ? renderCustomGridBody : undefined}
```

Comment:
@elastic/response-ops team, do we know why `dynamicRowHeight` triggers `renderCustomGridBody`. Dynamic height can simply be achieved by passing `height: auto` in rowHeightUtils.

Reply:
Okay I just found this PR : https://github.com/elastic/kibana/pull/169257 but i am not sure why how `customRenderGridBody` will improve the re-renders. Eui's `auto` height works perfect as shown in above benchmarks.

Reply:
Hey! Let's wait for @umbopepato input (he is on PTO). He is the most knowledgeable person regarding this.